proc-macros/src/rpc_macro.rs

```diff
@@ -81,11 +82,18 @@ impl RpcMethod {
 			.filter_map(|arg| match arg {
 				syn::FnArg::Receiver(_) => None,
 				syn::FnArg::Typed(arg) => match *arg.pat {
-					syn::Pat::Ident(name) => Some((name, *arg.ty)),
-					_ => panic!("Identifier in signature must be an ident"),
+					syn::Pat::Ident(name) => Some(Ok((name, *arg.ty))),
+					syn::Pat::Wild(_) => Some(Err(syn::Error::new(
+						span,
+						"Method argument names must be a valid Rust identifiers; got `_` instead",
```

Comment:
Not sure what the correct verbage would be in order to correctly express the error behind the syntax, but it sounds like to me it would be either:

"be a valid Rust identifier"

or

"be valid Rust identifiers"

Reply:
ty! :)